vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1956,6 +1977,11 @@ void vtkF3DRenderer::UpdateActors()
   {
     this->ConfigureGridUsingCurrentActors();
   }
+
+  if (!this->AxesActorConfigured && this->ModernAxisRepresentation)
+  {
+    this->ConfigureAxesActor();
+  }
```

Comment:
according to the comment above, it may be best to configure this before the grid.